### DISCUSSION TITLE
Storefront API calls?

### DISCUSSION THREAD

**Discussion by ignacio-dev** [Kyon147/laravel-shopify#322] [Q&A]
Is it possible to make calls using the Storefront Rest API using this library? So far I have only been successful at using the GraphQL and the Admin Rest.

**Comment by Kyon147** [comment]
Storefont API is through Javascript or another frontend language, where as the built in api helper for this package only deals with the admin api.

Why would you want to use the storefront api, as the admin api should have everything you need and more

**Comment by ignacio-dev** [comment]
I might be wrong (I'm new to Shopify) but I think Storefront API lets you order products by price, while the GraphQL does not?
